### PR TITLE
Initial Build

### PR DESCRIPTION
**Pull request by trisha0996**

### CODE REVIEW

**Review by yousefafc — APPROVED** [review]
LGTM

File: modules/subnets/main.tf [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "${x.subnet_region}/${x.subnet_name}" => x
+  }
+}
+
+resource "google_compute_subnetwork" "subnetwork" {
+  for_each                 = local.subnets
+  name                     = each.value.subnet_name
+  ip_cidr_range            = each.value.subnet_ip
+  region                   = each.value.subnet_region
+  private_ip_google_access = lookup(each.value, "subnet_private_access", "false")
```
Comment (niven01) [inline modules/subnets/main.tf]: Can we default private_ip_google_access to true

**Review by niven01 — CHANGES_REQUESTED** [review]
Can we default private_ip_google_access to true

File: modules/subnets/main.tf (hunk 1 above)
Comment (trisha0996) [reply modules/subnets/main.tf]: done

**Review by niven01 — APPROVED** [review]
LGTM

**Pull request merged**